scripts/ldflags.sh

```diff
@@ -13,6 +13,7 @@ PACKAGE="github.com/CircleCI-Public/circleci-yaml-language-server/pkg/utils"
 VERSION=$(cat ~/version)
 
 LDFLAGS=(
+  "-extldflags -static"
```

Comment:
I wonder how the whole "cross-compilation with `zig cc` and cgo and dynamically linking a libc for Windows but now we give `-static`" thing works. 🤔

This is just a question for my curiosity, not a problem with the code. Have you investigated? I'd be glad to learn more.

Reply:
I did try on Windows and it turns out it's not working BUT turns out the actual version doesn't work either... So I believe this will have to be handled in a separate ticket.